internal/engine/actions/remediate/pull_request/pull_request.go

```diff
@@ -40,6 +40,7 @@ import (
 	"github.com/mindersec/minder/internal/profiles/models"
 	"github.com/mindersec/minder/internal/util"
 	pb "github.com/mindersec/minder/pkg/api/protobuf/go/minder/v1"
+	interfaces2 "github.com/mindersec/minder/pkg/engine/v1/interfaces"
```

Comment:
this is irritating about goland, can you rename the import name? (You can use the refactor action on `interfaces2`)

Reply:
Will do